src/CalcManager/Ratpack/basex.cpp

```diff
@@ -36,29 +36,29 @@ void _mulnumx( PNUMBER *pa, PNUMBER b );
 //
 //----------------------------------------------------------------------------
 
-void __inline mulnumx( PNUMBER *pa, PNUMBER b )
+void __inline mulnumx( NUMBER *pa, NUMBER b )
 
 {
-    if ( b->cdigit > 1 || b->mant[0] != 1 || b->exp != 0 )
+    if ( b.cdigit > 1 || b.mant[0] != 1 || b.exp != 0 )
```

Comment:
Throws an exception for me in Debug x64 due to `b.mant[0]` but `b.size()` is 0.

Reply:
Immediately at startup? From which function was `mulnumx` called?

Debug x64 builds and runs fine on my machine.

Reply:
Let's table this and work off the most recent commit.